test/relations.jl

```diff
@@ -291,31 +291,32 @@ end
     T = FT(300)
     e_kin = FT(11)
     e_pot = FT(13)
-    @test air_temperature(param_set, _cv_d * (T - _T_0)) === FT(T)
+    @test air_temperature(param_set, _cv_d * (T - _T_0) - _R_d * _T_triple) ===
```

Comment:
Should it be _T_0 instead of _T_triple here and elsewhere? (The numbers are exactly the same now but I think it should be _T_0 is they are not the same.)

Reply:
Ugh, yes! Thank you. It's correct in the source code. I don't know why I used the triple point in tests. Will fix!